react-example/src/components/EventsForm.tsx

```diff
@@ -1,46 +1,27 @@
 import { FC, FormEvent, useState } from 'react';
-import styled from 'styled-components';
+import {
+  Button,
+  EndpointWrapper,
+  Form,
+  Heading,
+  Response
+} from '../views/Components.styled';
 import { IterablePromise, IterableResponse } from '@iterable/web-sdk';
 import TextField from 'src/components/TextField';
-import Button from 'src/components/Button';
 
 interface Props {
   endpointName: string;
   heading: string;
+  needsEventName?: boolean;
   method: (...args: any) => IterablePromise<IterableResponse>;
 }
 
-const Form = styled.form`
-  display: flex;
-  flex-flow: column;
-  width: 45%;
-
-  @media (max-width: 850px) {
-    width: 100%;
-  }
-`;
-
-const Response = styled.pre`
-  width: 45%;
-  white-space: break-spaces;
-
-  @media (max-width: 850px) {
-    width: 100%;
-  }
-`;
-
-const EndpointWrapper = styled.div`
-  display: flex;
-  flex-flow: row;
-  width: 100%;
-  justify-content: space-between;
-`;
-
-const Heading = styled.h2`
```

Comment:
this PR also has some cleanup. These styled components were re-used elsewhere so I just moved them to their own file and exported/imported